src/entities/Book/Book.schema.ts

```diff
@@ -0,0 +1,16 @@
+import * as mongoose from 'mongoose';
+import IBook from './Book';
+
+const bookSchema = new mongoose.Schema({
+  id: mongoose.Schema.Types.ObjectId,
```

Comment:
As far as I remember we should not assign manually id as it will be done automatically by MongoDB (so the id property should be deleted)